src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1386,4 +1396,53 @@ public static void onAdvancement(EntityPlayerMP player, Advancement advancement)
     {
         MinecraftForge.EVENT_BUS.post(new AdvancementEvent(player, advancement));
     }
+
+    /**
+     * Used as the default implementation of {@link Item#getCreatorModId}. Call that method instead.
+     */
+    @Nullable
+    public static String getDefaultCreatorModId(ItemStack itemStack)
```

Comment:
Add `@Nonnull` to the `ItemStack` param here?

Reply:
Good idea